Misc/NEWS.d/next/Library/2024-09-24-19-32-14.gh-issue-123014.zVcfkZ.rst

```diff
@@ -0,0 +1,3 @@
+:func:`os.pidfd_open` and :func:`signal.pidfd_send_signal` are now
```

Comment:
Do these methods need addition annotations in the docs to highlight that they're not available?

Reply:
Ah right, you can write "Android (API level 31 and newer)" in "Availability" of the modified functions.